pkg/cloudinary-jquery-file-upload/README.md

```diff
@@ -1,282 +1,109 @@
-[![Build Status](https://travis-ci.org/cloudinary/cloudinary_js.svg?branch=master)](https://travis-ci.org/cloudinary/cloudinary_js) [![npm](https://img.shields.io/npm/v/cloudinary-jquery-file-upload.svg?maxAge=2592000)]() [![Bower](https://img.shields.io/bower/v/cloudinary-jquery-file-upload.svg?maxAge=2592000)]() [![license](https://img.shields.io/github/license/cloudinary/pkg-cloudinary-jquery-file-upload.svg?maxAge=2592000)]()
+Cloudinary jQuery File Upload Plugin
+=======================================
 
-# Cloudinary jQuery Plugin with file upload - bower and npm repository
+## About
+The Cloudinary JQuery File Upload Plugin allows you to effortlessly upload assets to your cloud.
 
-Cloudinary is a cloud service that offers a solution to a web application's entire image management pipeline.
+#### Note
+This Readme provides basic installation and usage information.
+For the complete documentation, see the [Javascript SDK Guide](https://cloudinary.com/documentation/jquery_integration).
```

Comment:
```suggestion
For the complete documentation, see the [upload section of the jQuery SDK Guide](documentation/jquery_image_and_video_upload#direct_uploading_from_the_browser).
```

Reply:
Suggesting linking to the relevant section of the jQuery guide, since the entire guide covers more than just the upload plugin.

Reply:
Thanks @jackieros, the url you added already appears under the ##upload section, I thought it would be good to have this too under ##Note.

Reply:
"this too" = you mean you thought it makes sense here to link to the entire SDK guide? 
We can - thought the entire guide covers transformations etc, and this isn't the readme for the entire [jQuery SDK](https://github.com/cloudinary/pkg-cloudinary-jquery), it's just the readme for the file upload plugin, so not sure users will understand where they've arrived if they land on that guide from this readme?

But if you prefer to link them to the entire guide, that's fine - in that case, pls just fix the link text to [jQuery SDK Guide]  (currently says "Javascript")